internal/network/command_dns_forwarder_update.go

```diff
@@ -0,0 +1,90 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingdnsforwarderv1 "github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-dnsforwarder/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDnsForwarderUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update <id>",
+		Short: "Update an existing DNS forwarder.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.dnsForwarderUpdate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update the name of DNS forwarder "dnsf-123456".`,
+				Code: `confluent network dns forwarder update dnsf-123456 --name "my-new-dns-forwarder"`,
+			},
+			examples.Example{
+				Text: `Update the gateway and domains of DNS forwarder "dnsf-123456".`,
+				Code: `confluent network dns forwarder update dnsf-123456 --gateway gw-123456 --domains abc.com,def.com`,
+			},
+		),
+	}
+
+	cmd.Flags().String("name", "", "Name of the DNS forwarder.")
+	cmd.Flags().StringSlice("domains", nil, "A comma-separated list of domains for the DNS forwarder to use.")
+	cmd.Flags().String("gateway", "", "Gateway ID.")
```

Comment:
We do not support updates to gatewayID (at least not yet :-)). Let's remove this command-line argument for `update` command

Reply:
instead we should add support to update DNS_Server_Ips with `--dns-server-ips`